### PR TITLE
fix: Google connexion endpoint

### PR DESCRIPTION
**Pull request by b4chh**
#### Description

Google connexion 

#### Changelog

Implementation of the following endpoint :
```rpc AuthenticateGoogle(AuthenticateGoogleRequest) returns (AuthenticateGoogleResponse) ```

### CODE REVIEW

File: noted/accounts/v1/accounts.proto [hunk 1]
```diff
@@ -207,3 +215,14 @@ message UpdateAccountPasswordRequest {
...
     Account account = 1 [(google.api.field_behavior) = REQUIRED];
 }
+
+
+message AuthenticateGoogleRequest {
+    string code = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message AuthenticateGoogleResponse {
+    string password = 1 [(google.api.field_behavior) = REQUIRED];;
```
Comment (gabriel654165) [inline noted/accounts/v1/accounts.proto:225]: There is maybe too much ";" ahah

File: noted/accounts/v1/accounts.proto [hunk 2]
```diff
@@ -207,3 +215,14 @@ message UpdateAccountPasswordRequest {
 message UpdateAccountPasswordResponse {
     Account account = 1 [(google.api.field_behavior) = REQUIRED];
 }
+
+
+message AuthenticateGoogleRequest {
+    string code = 1 [(google.api.field_behavior) = REQUIRED];
+}
+
+message AuthenticateGoogleResponse {
```
Comment (gabriel654165) [inline noted/accounts/v1/accounts.proto:224]: Why do you return the informations of the account ? What about the jwtoken the client will need ? Look at the `AuthenticateResponse`

**Review by gabriel654165 — CHANGES_REQUESTED** [review]

File: noted/accounts/v1/accounts.proto (hunk 1 above)
Comment (b4chh) [reply noted/accounts/v1/accounts.proto:225]: You're right, I'll fix it

File: noted/accounts/v1/accounts.proto (hunk 2 above)
Comment (b4chh) [reply noted/accounts/v1/accounts.proto:224]: Thanks, I changed with a token

**Review by gabriel654165 — APPROVED** [review]

**Pull request merged**